### PR TITLE
Build docker image and introduce bundler

### PR DESCRIPTION
**Pull request by alexpop**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,29 @@
+PATH
```
Comment (chris-rock) [inline Gemfile.lock]: Lets not included the lockfile

**Comment by alexpop** [comment]
Good catch, thanks for reviewing it Chris

File: test/accuracy/test.rb [hunk 2]
```diff
@@ -66,12 +70,11 @@ def print_line(format_str, field, sslrb, testssl)
 
   res1 = {}
   res1time = Benchmark.measure {
-    cmd("rm *json; yes | ./#{testssl}/testssl.sh --json -p localhost:#{port}")
+    cmd("rm *json; yes | ./#{testssl}/testssl.sh --json -p 127.0.0.1:#{port}")
   }
   testssljson = Dir['*.json'][0]
   if testssljson.nil?
     puts 'No openssl output JSON found!!'
-    require "pry"; binding.pry
```
Comment (arlimus) [inline test/accuracy/test.rb]: Please fail here. Since we depend on testssljson a few lines further down ;)

**Comment by arlimus** [comment]
ty for the adjustments Alex :D

**Pull request merged**